frontend/cypress/support/applicationCommands.js

```diff
@@ -0,0 +1,72 @@
+import { CreateApplicationDocument, SubmitApplicationDocument } from "../../talentsearch/src/js/api/generated"
+import { UpdatePoolCandidateStatusDocument } from "../../admin/src/js/api/generated";
+
+function getGqlString(doc) {
+  return doc.loc && doc.loc.source.body;
+}
+
+Cypress.Commands.add('createApplication', (userId, poolId) => {
+  cy.request({
+    method: 'POST',
+    url: '/graphql',
+    auth: {
+      bearer: window.localStorage.getItem('access_token'),
+    },
+    body: {
+      operationName: 'createApplication',
+      query: getGqlString(CreateApplicationDocument),
+      variables: {
+        userId,
+        poolId
+      },
+    },
+  })
```

Comment:
Maybe package this into `cy.graphqlRequest()` and pack the method/url/auth in there?

Reply:
I've added it in https://github.com/GCTC-NTGC/gc-digital-talent/pull/4998/commits/10455e5728f9216afbbbde6532833e4ecefc0834.  It's blown out the size of the PR but I do like it!